src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -584,14 +583,12 @@ where
         };
 
         let table_ids_cnt = fragment_graph.table_ids_cnt;
-        let default_parallelism = if self.env.opts.minimal_scheduling {
```

Comment:
Do we need to get rid of `minimal_scheduling` config now? Not sure what the hierarchy of configurations should be? Btw, what is this `minimal_scheduling` config for in the first place?

Reply:
Ok, I see, user will have to set the session variable.